rpy2/robjects/help.py

```diff
@@ -262,10 +262,10 @@ def usage(self) -> str:
         """ Get the usage for the object """
         return self._get_section(r'\usage')
 
-    def iteritems(self):
+    def items(self):
```

Comment:
It is fine to add a method `.items()` but `.iteritems()` should remain with a deprecation warning, even if merely a wrapper for the method `.items()`. Without this, code relying on this part of the API would immediately break.

Reply:
Good point. Just pushed a commit to revive the `iteritems`.  (identical method as `items`.)

I also changed both `items()` and `iteritems()` to return the output of `self.sections.items()` rather than `self.sections.items`. (this seems to be another bug? if not please revert.)